encoder_test.go

```diff
@@ -51,7 +51,7 @@ func TestFilled(t *testing.T) {
 	valExists(t, "f07", "seven", vals)
 	valExists(t, "f08", "8", vals)
 	valExists(t, "f09", "1.618000", vals)
-	valExists(t, "F12", "12", vals)
```

Comment:
(What drives removal of this test? Want to make sure there isn’t a behavior change for existing users)

Reply:
Key will be renamed from "F12"  to "F11.F12" as F12 is under F11 struct is that test. Decoding in that case would expect "F11.F12" as a key as well. I am afraid that pull request amends how encoding works so unfortunately there will be a behavior change for existing users :(